areas/areas.go

```diff
@@ -105,6 +103,34 @@ func (c *Client) GetArea(ctx context.Context, userAuthToken, serviceAuthToken, c
 	return
 }
 
+// GetRelations gets the child areas
+func (c *Client) GetRelations(ctx context.Context, userAuthToken, serviceAuthToken, collectionID, areaID, acceptLang string) (relations []Relation, err error) {
+	uri := fmt.Sprintf("%s/v1/areas/%s/relations", c.hcCli.URL, areaID)
+	clientlog.Do(ctx, "retrieving area relations", service, uri)
+	// Do request
+	res, err := c.doGetWithAuthHeaders(ctx, userAuthToken, serviceAuthToken, collectionID, uri, nil, "", "en")
+	if err != nil {
+		return
+	}
+	defer closeResponseBody(ctx, res)
+
+	if res.StatusCode != http.StatusOK {
+		err = NewAreaAPIResponse(res, uri)
+		return
+	}
+
+	b, err := ioutil.ReadAll(res.Body)
+	if err != nil {
+		return
+	}
+	var body []Relation
+	if err = json.Unmarshal(b, &body); err != nil {
```

Comment:
Is unmarshalling here necessary as you're unmarshalling the response body below too?

Reply:
Sorry, my bad